tests/geometry/rotation.rs

```diff
@@ -206,7 +206,7 @@ mod proptest_tests {
             prop_assert!(relative_eq!(v2.angle(&vv2), 0.0, epsilon = 1.0e-7));
             prop_assert!(relative_eq!(v3.angle(&vv3), 0.0, epsilon = 1.0e-7));
             prop_assert!(relative_eq!(Rotation2::rotation_between(&v2, &vv2), Rotation2::identity()));
```

Comment:
I think this code should also get the epsilon that the code below gets or is there a reason why it would work exactly with a factor of `4.23` and not with `3.42`?

Reply:
Done: rebased on current `main`, added `epsilon = 1.0e-7` for consistency.